src/redux/modules/posts.js

```diff
@@ -146,9 +154,18 @@ const reducer = (state = initialState, action) => {
         loading: false,
         error: action.error,
       };
+    case SELECT_POST:
+      return {
+        ...state,
+        loading: true,
+        error: null,
```

Comment:
`error` should usually only be used with async actions. No need to reset it here (you should reset it when you dispatch an async action)

Reply:
the same thing with `loading`, right?

Reply:
Yes